examples/kubernetes.json

```diff
@@ -29,6 +29,9 @@
     "servicePrincipalProfile": {
       "clientId": "",
       "secret": ""
+    },
+    "featureFlags": {
+      "enableTelemetry": true

```

Comment:
I think we want to revert this change to the main example file before the PR is merged

Reply:
I left it there intentionally, but in hindsight, I think you are right. I'll remove it.